pkg/controller/podgc/gc_controller.go

```diff
@@ -200,7 +205,7 @@ func (gcc *PodGCController) gcTerminated(pods []*v1.Pod) {
 		wait.Add(1)
 		go func(namespace string, name string) {
 			defer wait.Done()
-			if err := gcc.deletePod(namespace, name); err != nil {
+			if err := gcc.deletePod(namespace, name, v1.GarbageCollectedTerminated); err != nil {
```

Comment:
We don't need this condition. The pod already terminated on its own. It's not a disruption.